tester_app/app/assets/stylesheets/_bootstrap-custom.scss

```diff
@@ -0,0 +1,55 @@
+// This file is used instead of
+// @import 'boostrap';
+// To allow for customization of what parts of bootstrap are imported.
+// Refer to this file: https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/_bootstrap.scss
+
+// Core variables and mixins
+@import 'bootstrap/variables';
+@import 'bootstrap/mixins';
```

Comment:
still missing the boostrap-variable-customization file that the JS loader file uses.